pandas/core/strings.py

```diff
@@ -439,8 +439,7 @@ def str_endswith(arr, pat, na=np.nan):
 def str_replace(arr, pat, repl, n=-1, case=None, flags=0, regex=True):
     r"""
     Replace occurrences of pattern/regex in the Series/Index with
-    some other string. Equivalent to :meth:`str.replace` or
-    :func:`re.sub`.
+    some other string. Equivalent to :func:`re.sub`.
```

Comment:
@gsganden, somehow unrelated to your change, but the docstring should start with a one-liner (see [1]). Do you think you can summarize what the method does, and then add the detailed explanation you're proposing in the extended summary?

Also, it'd be great if you can fix the rest of the docstring based on the standard defined in [1].


1. http://pandas-docs.github.io/pandas-docs-travis/contributing_docstring.html#section-1-short-summary